src/main/pages/workspace-admin/volumes.adoc

```diff
@@ -8,17 +8,19 @@ folder: workspace-admin
 ---
 
 
-[id="default-volumes"]
-== Default Volumes
+[id="default-volumes_for_workspace_containers"]
+== Default volumes for workspace containers
 
-By default workspace containers/pods start with one default volume/PVC that persists `/projects` where workspace projects are physically located. When a workspace is stopped its machines are destroyed, however, volumes stay there.
+By default, workspace containers start with a default volume and have a minimum of one PVC that is located in `/projects` directory. 
 
-[id="user-provided-volumes"]
-== User-Provided Volumes
+Workspace projects are physically located in the `/projects` directory. When a workspace stops, the machines are destroyed, but the volumes persist.
 
-Your workspace may need additional volumes though, say, to persist a local Maven repo, node_modules, ruby jems, authorized_keys for ssh connections etc. You can add additional volumes for your workspace machines - each machine can get as many volumes as an underlying infrastructure can afford, or depending on imposed limits (that’s mostly true for OpenShift).
+[id="Adding-volumes"]
+== Adding volumes
 
-Volumes can be added either in User Dashboard or directly in machine configuration:
+Your workspace may need additional volumes if your data must persist for a local Maven repository, node_modules folder, ruby jems, or authorized_keys file for SSH connections. You can add additional volumes for your workspace machines. Each machine can add as many volumes as the underlying infrastructure can support. OpenShift may impose a limit on the number of volumes.
```

Comment:
Typos/need styling: `node_modules` directory (not folder), Ruby gems, `authorized_keys` file.